css/style.css

```diff
@@ -1,5 +1,6 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,700;0,900;1,900&display=swap");
-
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,700);
```

Comment:
use the default font itself so that the load is less.

Reply:
Done.